docs/overview.rst

```diff
@@ -10,7 +10,7 @@ This package is meant for users who don't care how the code works necessarily, b
 As a result this package is meant to be as easy-to-use as possible, coupled with extensive documentation of all functions and standard examples of how to use them sensibly.
 
 The core Python infrastructure is influenced by, and extends the functionaity of, the `Astropy <http://astropy.org>`__ package, a superb set of tools for astrophysical analysis, primarily for FITS images.
-Additionally, much of the methodology has been derived from, and augmented by, the `LIGO Algorithm Library Suite (LALSuite) <https://www.lsc-group.phys.uwm.edu/daswg/projects/lalsuite.html>`_, a large collection of primarily ``C99`` routines for analysis and manipulation of data from gravitational-wave detectors.
+Additionally, much of the methodology has been derived from, and augmented by, the `LVK Algorithm Library Suite (LALSuite) <https://www.lsc-group.phys.uwm.edu/daswg/projects/lalsuite.html>`_, a large collection of primarily ``C99`` routines for analysis and manipulation of data from gravitational-wave detectors.
```

Comment:
@dbkeitel, thanks for adding this. I don't suppose you can update the URL as well, since the old one 404s.

Reply:
Good catch, I didn't check it. Let me put in the DOI link instead.